### DISCUSSION TITLE
Andorid Explore Filter Bug

### DISCUSSION THREAD

**Discussion by Linyoux** [community/community#139558] [Projects and Issues] [answered]
Some starred repositories are still being pushed after I unstarred them. After changing the filter and changing it back, the pushes of other starred repositories are gone, leaving only the people I am following, and even after unfollowing, the pushes continue!!!

**Comment by VadymMakohon** [comment, marked as answer]
Hello, @ZKENSS!

I would recommend you to try the following steps in order to solve this issue.
Try to test it on different browsers and see if the problem is still the same.  If it doesn't happen on another browser, the issue could be browser-specific. Try to clear the cache and cookies of the browser you are using, or try accessing GitHub in incognito/private mode. 

You are writing that the pushes of other starred repositories are gone. Did you try to manually review your following and starred repositories settings? Go to Your Stars and ensure the repositories have indeed been unstarred. After that see the Following section under Your Profile to confirm you’re not mistakenly following repositories or users you’ve unfollowed. in addition try explicitly refreshing or resetting the filter on GitHub’s starred repositories page to see if it forces an update of the data.

If the issue persists after trying these steps, you can contact GitHub support with specific details about what is happening.

If this response was helpful, I’d appreciate it if you could mark it as the answer. Thanks!

**Reply by Linyoux** [reply]
Oh, thank you! Because the filter on the Android version of GitHub doesn't have a reset feature, I couldn't restore it to its original state. So I reset it on the web version, and it's back to normal now.

**Reply by VadymMakohon** [reply]
Hey, cool. I am glad it helped 👍
